scripts/release.mjs

```diff
@@ -50,10 +50,14 @@ CURRENT_BRANCH = CURRENT_BRANCH.trim();
 let { stdout: DEFAULT_BRANCH } = await $`git remote show origin | sed -n '/HEAD branch/s/.*: //p'`;
 DEFAULT_BRANCH = DEFAULT_BRANCH.trim();
 
+const crypto = require('crypto');
```

Comment:
You should be able to use `import crypto from 'crypto';` right?

Reply:
@westeezy, yes, thanks for the catch! I just pushed the changes up.